packages/matchbox/src/components/Modal/Content.js

```diff
@@ -7,7 +7,7 @@ const Content = React.forwardRef(function Content(props, ref) {
   const { children } = props;
 
   return (
-    <Box data-id="modal-content" p="500" ref={ref}>
+    <Box data-id="modal-content" p="500" maxHeight="60vh" overflow="scroll" ref={ref}>
```

Comment:
I would change this to overflowY so the horizontal scrollbar are hidden if someone has them enabled

![Screen Shot 2020-08-12 at 9 02 57 AM](https://user-images.githubusercontent.com/3903325/90018294-b2140a00-dc7a-11ea-93b5-3d928a3ee8ae.png)